cypress/e2e/specialFeatures/cookieBanner/index.cy.js

```diff
@@ -1,54 +1,169 @@
-import ramdaPath from 'ramda/src/path';
-import config from '../../../support/config/services';
-import describeForEuOnly from '../../../support/helpers/describeForEuOnly';
```

Comment:
Since this is no longer needed, the whole file can be deleted https://github.com/bbc/simorgh/blob/3b259452259ef0ca9d93a30300dbbee2f0a60924/cypress/support/helpers/describeForEuOnly.js